diff --git "a/src/map/time_server.cpp" "b/src/map/time_server.cpp"
index 514fbd3f26..f6a23c3571 100644
--- "a/src/map/time_server.cpp"
+++ "b/src/map/time_server.cpp"
@@ -94,7 +94,7 @@ int32 time_server(time_point tick, CTaskMgr::CTask* PTask)
     {
         if (tick > (lastTickedJstMidnight + 1h))
         {
-            if (settings::get<bool>("main.ENABLE_DAILY_TALLY"))
+            if (settings::get<bool>("main.ENABLE_DAILY_TALLY") && map_port == 54230)
             {
                 daily::UpdateDailyTallyPoints();
             }
