diff --git "a/src/map/entities/battleentity.h" "b/src/map/entities/battleentity.h"
index b804caaa77..a8a3c4799e 100644
--- "a/src/map/entities/battleentity.h"
+++ "b/src/map/entities/battleentity.h"
@@ -210,7 +210,8 @@ enum SUBSKILLTYPE
     SUBSKILL_RAPHIE     = 65,
     SUBSKILL_MAC        = 66,
     SUBSKILL_SILAS      = 67,
-    SUBSKILL_TOLOI      = 68
+    SUBSKILL_TOLOI      = 68,
+    SUBSKILL_DSANCHEZ   = 100
 };
 
 // ячейки экипировки. монстры используют лишь первые четыре, персонаж использует все
diff --git a/src/map/lua/lua_baseentity.cpp b/src/map/lua/lua_baseentity.cpp
index 10211143373..6efd3050ca5 100644
--- a/src/map/lua/lua_baseentity.cpp
+++ b/src/map/lua/lua_baseentity.cpp
@@ -12490,7 +12490,7 @@ bool CLuaBaseEntity::hasValidJugPetItem()
 
     CItemWeapon* PItem = static_cast<CItemWeapon*>(static_cast<CCharEntity*>(m_PBaseEntity)->getEquip(SLOT_AMMO));
 
-    if (PItem != nullptr && PItem->getSubSkillType() >= SUBSKILL_SHEEP && PItem->getSubSkillType() <= SUBSKILL_TOLOI)
+    if (PItem != nullptr && PItem->getSubSkillType() > 0 && ((PItem->getSubSkillType() >= SUBSKILL_SHEEP && PItem->getSubSkillType() <= SUBSKILL_TOLOI) || (PItem->getSubSkillType() == SUBSKILL_DSANCHEZ)))
     {
         return true;
     }
diff --git a/src/map/utils/petutils.h b/src/map/utils/petutils.h
index c01b4163b29..9be967a16ee 100644
--- a/src/map/utils/petutils.h
+++ b/src/map/utils/petutils.h
@@ -58,7 +58,7 @@ enum PETID
     PETID_CHOCOBO            = 74,
     PETID_LUOPAN             = 75,
     PETID_SIREN              = 76,
-    MAX_PETID                = 77,
+    MAX_PETID                = 101,
 };
 
 class CBattleEntity;