 src/map/utils/charutils.cpp | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/src/map/utils/charutils.cpp b/src/map/utils/charutils.cpp
index 3e25c61be2..d8caca4c8f 100644
--- a/src/map/utils/charutils.cpp
+++ b/src/map/utils/charutils.cpp
@@ -3720,19 +3720,19 @@ namespace charutils
                                 exp *= 1.00f;
                                 break;
                             case 2:
-                                exp *= 0.90f;
+                                exp *= 1.00f;
                                 break;
                             case 3:
-                                exp *= 0.80f;
+                                exp *= 1.00f;
                                 break;
                             case 4:
-                                exp *= 0.70f;
+                                exp *= 1.00f;
                                 break;
                             case 5:
-                                exp *= 0.60f;
+                                exp *= 1.00f;
                                 break;
                             case 6:
-                                exp *= 0.50f;
+                                exp *= 1.00f;
                                 break;
                             default:
                                 exp *= (1.8f / pcinzone);
@@ -3748,19 +3748,19 @@ namespace charutils
                                 exp *= 1.00f;
                                 break;
                             case 2:
-                                exp *= 0.90f;
+                                exp *= 1.00f;
                                 break;
                             case 3:
-                                exp *= 0.80f;
+                                exp *= 1.00f;
                                 break;
                             case 4:
-                                exp *= 0.70f;
+                                exp *= 1.00f;
                                 break;
                             case 5:
-                                exp *= 0.60f;
+                                exp *= 1.00f;
                                 break;
                             case 6:
-                                exp *= 0.50f;
+                                exp *= 1.00f;
                                 break;
                             default:
                                 exp *= (1.8f / pcinzone);
@@ -3775,19 +3775,19 @@ namespace charutils
                                 exp *= 1.00f;
                                 break;
                             case 2:
-                                exp *= 0.80f;
+                                exp *= 1.00f;
                                 break;
                             case 3:
-                                exp *= 0.65f;
+                                exp *= 1.00f;
                                 break;
                             case 4:
-                                exp *= 0.50f;
+                                exp *= 1.00f;
                                 break;
                             case 5:
-                                exp *= 0.45f;
+                                exp *= 1.00f;
                                 break;
                             case 6:
-                                exp *= 0.40f;
+                                exp *= 1.00f;
                                 break;
                             default:
                                 exp *= (1.8f / pcinzone);
