diff --git "a/src/map/ai/controllers/trust_controller.cpp" "b/src/map/ai/controllers/trust_controller.cpp"
index 5dbb377be9..063b0045c3 100644
--- "a/src/map/ai/controllers/trust_controller.cpp"
+++ "b/src/map/ai/controllers/trust_controller.cpp"
@@ -37,6 +37,7 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include "../../status_effect_container.h"
 #include "../../utils/charutils.h"
 #include "../ai_container.h"
+#include "../../notoriety_container.h"
 
 CTrustController::CTrustController(CCharEntity* PChar, CTrustEntity* PTrust)
 : CMobController(PTrust)
@@ -277,7 +278,7 @@ void CTrustController::DoRoamTick(time_point tick)
     }
 
     if (POwner->CanRest() && m_Tick - POwner->LastAttacked > m_tickDelays.at(0) && m_Tick - m_CombatEndTime > m_tickDelays.at(0) &&
-        m_Tick - m_LastHealTickTime > m_tickDelays.at(m_NumHealingTicks))
+        m_Tick - m_LastHealTickTime > m_tickDelays.at(m_NumHealingTicks) && !POwner->PMaster->PNotorietyContainer->hasEnmity())
     {
         if (POwner->health.hp != POwner->health.maxhp || POwner->health.mp != POwner->health.maxmp)
         {
diff --git "a/src/map/ai/controllers/trust_controller.h" "b/src/map/ai/controllers/trust_controller.h"
index bcc1930e0f..684df09ba1 100644
--- "a/src/map/ai/controllers/trust_controller.h"
+++ "b/src/map/ai/controllers/trust_controller.h"
@@ -74,7 +74,7 @@ private:
 
     time_point                        m_CombatEndTime;
     time_point                        m_LastHealTickTime;
-    std::vector<std::chrono::seconds> m_tickDelays      = { 15s, 10s, 10s, 3s };
+    std::vector<std::chrono::seconds> m_tickDelays      = { 19s, 12s, 12s, 6s };
     std::size_t                       m_NumHealingTicks = { 0 };
 
     time_point m_LastRangedAttackTime;
